sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/BigtableIO.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.io.bigtable;
+package org.apache.beam.sdk.io.gcp;
```

Comment:
yes -- specifically, I'd like there to be an `io.gcp` package that has most of the GCP-related files from `io`. I'd still like `bigtable` to be in its own package, so following the other ordering that would end up in `io.bigtable` -> `io.gcp.bigtable`.